README.md

```diff
@@ -869,6 +872,16 @@ this._viewer.setCurrentPage(4).then((success) => {
 });
 ```
 
+##### getDocumentPath
+Return the path of the current document.
+
+Return a Promise.
+
+```js
+this._viewer.getDocumentPath((path) => {
+  console.log('The path to current document is: ', path);
```

Comment:
this._viewer.getDocumentPath((path) => {
        console.log('The path to current document is: ', path);
      });

Reply:
and it doesnt seem to be working for me for some reason when i tried as is... not too sure why, could you check again?